src/index.ts

```diff
@@ -67,6 +67,12 @@ export type StylePropertySpecification = {
     expression?: ExpressionSpecificationDefinition;
     transition: boolean;
     default?: VariableAnchorOffsetCollectionSpecification;
+} | {
+    type: 'projectionDefinition';
+    'property-type': ExpressionType;
+    expression?: ExpressionSpecificationDefinition;
+    transition: boolean;
+    default?: string;
```

Comment:
How would you represent a state "between" projections using just string?

Reply:
I think the current default value is "mercator"

Reply:
No, the default should be `mercator`. I just don't think the type of it should be `string`, but instead be `ProjectionDefinition` or `ProjectionDefinitionSpecification`.

Reply:
it's ProjectionDefinitionSpecification now